### DISCUSSION TITLE
Week 13

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#244] [General]
This has been such a fun, challenging semester and I'm so proud of the progress my classmates and I have made. At the beginning of the semester, this course felt daunting and intimidating, and while I wouldn't necessarily feel confident applying for web design jobs, I feel extremely excited by the new knowledge I have gained. So many times while working through the lessons on FreeCodeCamp I would resort to a Google or YouTube search when I was feeling stuck and the results were often so helpful and enlightening. I told myself that even if I didn't fully understand something, if I just kept practicing eventually the repeated exposure would work out for me in the end, and it did! I have thought about going back through the FreeCodeCamp lessons and releasing my own batch of instructional videos on how to complete the lesson because I found them so helpful for myself and I noticed a lack of female participation in the space. Here's the first video I did:  [https://www.youtube.com/watch?v=SAWQytg7MCY&t=4s](url)

For my final project, I decided to publish it [https://audreyhki.github.io/A_Kienow-Project/](url) and I hope you check it out and let me know what you think! From my humble beginnings of knowing absolutely nothing about JS, HTML, or CSS, not knowing the purpose that each of the served, or how websites were built, I feel like a million bucks! As an ITLS-4265 student, I believe my final project shows a basic understanding of CSS and an intermediate level of HTML. I was able to add a link to the business's Instagram and well as include the personal logos that were given to me. Overall I think the site looks great. 

**Note: For some reason the links are not working in Github, but if you copy and paste them into a new tab they work just fine.

**Comment by Skye404** [comment]
The color palette is amazing. I love what you've done here. It's simple and easy to follow. Love it! Nice work.

**Comment by coopsand** [comment]
Great job! I love that the logo is centered with the menu at the top. It feels very organized.

**Comment by kassyoveson** [comment]
Very well done. I love the colors that you used, and your logo is really cool. It is well organized and easy to read. You even put the logo into the headers, which was probably not very easy. Good job! Is beefslayers a real restaurant?

**Reply by audreyhki** [reply]
Yes! Beefslayers is a real food vendor! They're just barely starting off by doing random events sort of like a food truck, you can see updates on where they'll be next on Instagram!

**Comment by gabebott** [comment]
This is super cool! I love that you saw a missing representation in coding education. That is really cool that you went from struggling to making videos to help others. I also really like your site and I think the graphics and the accessibility of the colors make it a really great finished product! Great job!
